pom.xml

```diff
@@ -8,7 +8,7 @@
   <!-- PROJECT NAME -->
   <name>Ewon Flexy Extensions Library</name>
   <!-- PROJECT VERSION -->
-  <version>1.15.14</version>
+  <version>1.16.0-pre2</version>
```

Comment:
We haven't been committing pre-releases to main. If this is not ready to be a full release this commit can be dropped.

Reply:
If you are doing this for maven reasons i think Alex generally just pushes a separate branch and tags it.

Reply:
It's common for repositories to have the main branch represent the bleeding edge of development and recommend a release branch/tag for releases.  Updating the version tag in the pom is useful because `mvn deploy` will update the local maven repository and then local connector development/testing can use this development version.

Reply:
will drop this commit

Reply:
Our main branch should represent the bleeding edge of development (in most cases), though we have just historically excluded pre-release tagging commits. The code/content commits themselves definitely belong in the main branch, but any tagged pre-release commits are typically left out and pushed+tagged directly to save time.